CONTRIBUTING.md

```diff
@@ -1,20 +1,69 @@
-# Contribution Guide
+# Contributing to AdoptOpenJDK/infrastructure
 
-The infrastructure project contains:
+Thank you for your interest in AdoptOpenJDK/infrastructure!
+
+We welcome and encourage all kinds of contributions to the project, not only
+code. This includes bug reports, user experience feedback, assistance in
+reproducing issues and more.
+
+The AdoptOpenJDK/infrastructure project contains:
 
 1. The [Ansible Playbooks](ansible/playbooks) for bootstrapping the build and test hosts (including a way to test Ansible)
 1. The overriding [Documentation](docs) for the build farm
 
-To make a change please:
+
+## Submitting a contribution to AdoptOpenJDK/infrastructure9
+
+You can propose contributions by sending pull requests (PRs) through GitHub.
+Following these guidelines will help us merge your pull requests smoothly:
+
+1. Your pull request is an opportunity to explain both what changes you'd like
+   pulled in, but also _why_ you'd like them added. Providing clarity on why
+   you want changes makes it easier to accept, and provides valuable context to
```

Comment:
I might add a bit saying that it may not be necessary to include all of this directly in the pull request if there is a linked issue with the information

Reply:
modified slightly with that in mind